packages/typegpu/tests/rawFn.test.ts

```diff
@@ -264,26 +264,28 @@ struct fragment_Output {
     }
 
     @fragment
-    fn fragment(input: fragment_Input) -> @location(0) vec4f {
+    fn fragment(in: fragment_Input) -> @location(0) vec4f {
       return vec4f(1.0f);
     }`),
     );
   });
 
   it('automatically adds struct definitions of argument types when resolving wgsl-defined functions', () => {
-    const Point = d.struct({
-      a: d.u32,
-      b: d.u32,
-    });
+    const Point = d
+      .struct({
+        a: d.u32,
+        b: d.u32,
+      })
+      .$name('Point');
 
     const func = tgpu['~unstable']
       .fn(
         [d.vec4f, Point],
         undefined,
       )(/* wgsl */ `(a: vec4f, b: Point) {
-    var newPoint: Point;
-    newPoint = b;
-  }`)
+        var newPoint: Point;
+        newPoint = b;
+      }`)
       .$name('newPointF');
```

Comment:
I think we should add `$uses()` here, since we're counting on the name of the struct to be `Point` exactly, and that only applies when in `strict` naming scheme.

Reply:
but then we wouldn't be checking, if Point struct is added to externals automatically. I will remove the usage in the function block

Reply:
and I assume that's a comment to the test with record argTypes below? cause with the approach with array args we don't need the name to be exact